vignettes/building-the-dvdrental-docker-image.Rmd

```diff
@@ -0,0 +1,88 @@
+---
+title: "Building the `dvdrental` Docker Image"
```

Comment:
Good job on this vignette. It is helpful that you walk through the reasoning of why and how the different files are used.